src/app/component/UsersAutocomplete.jsx

```diff
@@ -31,39 +32,43 @@ const SearchUsers = ({ editable, handleChange, selected, users }) => {
   };
   const options = users?.reduce(reducer, []) || [];
   const optionSelected = options.find((option) => option && selected && option.id === selected.id);
-  const defaultValue = editable ? optionSelected : "";
-  const value = !editable ? optionSelected : "";
+  const defaultValue = editable ? optionSelected : null;
 
   return (
-    <Autocomplete
-      id="user-search"
-      noOptionsText="No users found"
-      options={options}
-      classes={{ root: classes.root }}
-      getOptionLabel={(user) => user.searchString}
-      onChange={(event, newValue) => handleChange(newValue)}
-      value={value}
-      defaultValue={defaultValue}
-      renderInput={(params) => (
-        <TextField
-          {...params}
-          className={classes.textField}
-          label="Search a user..."
-          placeholder="search user with either name or phone number"
-          variant="outlined"
-          InputProps={{ ...params.InputProps, type: "search" }}
+    <>
+      {editable ? (
```

Comment:
instead of wrapping this whole component in the ternary I think it would be a lot more clear to just put an if statement before hand
```js
if (!editable) {
   return <H6>...
}

return <Autocomplete ...
```

Reply:
Done!